<?xml version='1.0'?>
<launch>
	<arg name="model" default="$(find robin_gazebo)/urdf/robin_hex.xacro"/>
	<arg name="world" default="robin_hex"/>
	<arg name="frame_id" default="map"/>
	<arg name="rviz" default="false"/>
	<arg name="rqt" default="false"/>

	<param name="/robin_gazebo/prop_layout" value="hex_x6"/>
	<param name="/robin_gazebo/base_arm_length" value="0.250"/>

	<group if="$(arg rviz)">
		<node name="rviz" pkg="rviz" args="-d $(find robin_gazebo)/config/display.rviz" type="rviz" respawn="false" output="screen"/>
	</group>

	<group if="$(arg rqt)">
		<node name="rqt" pkg="rqt_gui" type="rqt_gui" args="--perspective-file $(find robin_gazebo)/config/dashboard.perspective" respawn="false" output="screen"/>
	</group>

	<!-- Gazebo -->
	<include file="$(find robin_gazebo)/launch/robin_sim_gazebo.launch">
		<arg name="model" value="$(arg model)" />
		<arg name="world" value="$(arg world)" />
	</include>
</launch>
